src/app/pages/charging-stations/charging-station/parameters/charge-point/charging-station-charge-point.component.html

```diff
@@ -37,7 +37,7 @@ <h4 class="card-title text-left">
         <div class="row">
           <div class="col-md-6" *ngFor="let connectorID of chargePoint?.connectorIDs">
             <app-charging-station-connector [chargingStation]="chargingStation" [chargePoint]="chargePoint"
-              [connector]="chargingStation?.connectors[connectorID - 1]" [formConnectorsArray]="formConnectorsArray"
+              [connector]="findConnector(connectorID)" [formConnectorsArray]="formConnectorsArray"
```

Comment:
You broke one golden rule: don't use a method here!

Reply:
Fixed